#request

string filename

---

sensor_msgs/Image detect_result_image
lane_detect/LaneDeectResult detect_result_quality
